### PR TITLE
Amplify mutation implementation

### PR DESCRIPTION
**Pull request by sebastiancrossa**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,4 +1,4 @@
 SKIP_PREFLIGHT_CHECK=true
-CONSUMER_KEY=
-CONSUMER_SECRET=
-CALLBACK_URL=
\ No newline at end of file
+CONSUMER_KEY="1IsryRNkS3BjSKRfY6LqlJIO8"
```
Comment (barronwei) [inline .env]: 👀 We should remove these.

File: src/Auth.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import TwitterLogin from "react-twitter-login";
 const Auth = ({ addUser }) => {
   const authHandler = (error, data) => {
     if (error) {
-      alert("Please log into Twitter!");
+      alert(error);
```
Comment (barronwei) [inline src/Auth.js:7]: Alerting to the error to the user would not be helpful here. If everything is correctly configured on our end, the only reason that this should fail is the user did not authenticate.

File: .env (hunk 1 above)
Comment (sebastiancrossa) [reply .env]: Consumer key has changed since then, so we're all good!

File: src/Auth.js (hunk 2 above)
Comment (sebastiancrossa) [reply src/Auth.js:7]: Yes, you are totally right. I changed it for testing purposes but will revert it back to the next PR.

Thank you for noticing!

**Pull request merged**